libs/ledger-live-common/src/exchange/swap/hooks/useUpdateMaxAmount.ts

```diff
@@ -32,16 +32,21 @@ export const useUpdateMaxAmount = ({
   const [isMaxLoading, setIsMaxLoading] = useState(false);
 
   const toggleMax: SwapTransactionType["toggleMax"] = useCallback(
-    () =>
+    account =>
       setIsMaxEnabled(previous => {
         const next = !previous;
         if (previous) {
           setFromAmount(ZERO);
           setIsMaxLoading(false);
         }
+        let additionalFees;
+        if (account?.type === "Account" && account?.unit?.code === "ETH") {
+          additionalFees = new BigNumber(10000000000000000); // 0,01 ETHs
+        }
         bridge.updateTransaction(tx => ({
           ...tx,
```

Comment:
Does 'tx' have 'fromAccount' attribute? I do not think you need to pass any parameter to toggleMax

Reply:
Transaction is undefined in this context. I had to pass it

Reply:
> Transaction is undefined in this context. I had to pass it

If 'tx' if undefined in this context, can we remove it? (I can not understand how a bridge which update transaction can have transaction undefined)